devfiles/che4z/devfile.yaml

```diff
@@ -13,10 +13,13 @@ components:
     type: chePlugin
     id: broadcommfd/explorer-for-endevor/latest
   - 
-    id: eclipse/che-machine-exec-plugin/7.3.0
     type: chePlugin
+    id: broadcommfd/debugger-for-mainframe/latest
   - 
-    id: eclipse/che-theia/7.3.0
+    type: chePlugin
+    id: broadcommfd/hlasm-language-support/latest    
+  - 
+    id: eclipse/che-theia/latest
```

Comment:
If speccing for 7.3.0 is no longer necessary, what do you think about removing a strict editor definition? By default, nightly builds of Che will use `eclipse/che-theia/latest`, and release builds of Che will use e.g. `eclipse/che-theia/7.6.0` (which would be `latest` for the `7.6.0` release), so including che-theia here shouldn't be required.

Reply:
+1 to Angel's proposal, it's good thing to do and I believe all devfiles in registry miss editor definition, if no - it should be fixed.

Reply:
I agree, since the version is no longer needed. I've checked on my test environment and removed the strict editor version definition.